this.js

```diff
@@ -0,0 +1,9 @@
+const person_this = {
+    firstName: "John",
+    lastName: "Doe",
+    id: 5566,
+    fullName : function() {
+      return this.firstName + " " + this.lastName;
+    }
+  };
```

Comment:
we can use this operator in lot of places based on the use case, please practice more on this operator

Reply:
sure, I will.